oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/source/DatabaseService.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.server.core.source;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * @Author: Liu-Haoyang
+ */
+public class DatabaseService extends Source {
+
+    @Override
+    public Scope scope() {
+        return Scope.DatabaseService;
+    }
+
+    @Override
+    public String getEntityId() {
+        return String.valueOf(id);
+    }
+
+    @Getter @Setter private long id;
+    @Getter @Setter private String name;
+    @Getter @Setter private String statement;
+    @Getter @Setter private int componentId;
```

Comment:
Component Id had two meanings, contains: client side component id and server side component id, you must define a clear name.

Reply:
I agree. And consider to use these two
- DatabaseTypeId
- DatabaseTypeName